tests/sentry/api/test_event_search.py

```diff
@@ -768,271 +768,801 @@ def test_empty_string(self):
         assert parse_search_query("") == []
 
 
-class ParseBooleanSearchQueryTest(unittest.TestCase):
+class ParseBooleanSearchQueryTest(TestCase):
     def setUp(self):
         super(ParseBooleanSearchQueryTest, self).setUp()
-        self.term1 = SearchFilter(
-            key=SearchKey(name="user.email"),
-            operator="=",
-            value=SearchValue(raw_value="foo@example.com"),
-        )
-        self.term2 = SearchFilter(
-            key=SearchKey(name="user.email"),
-            operator="=",
-            value=SearchValue(raw_value="bar@example.com"),
-        )
-        self.term3 = SearchFilter(
-            key=SearchKey(name="user.email"),
-            operator="=",
-            value=SearchValue(raw_value="foobar@example.com"),
-        )
-        self.term4 = SearchFilter(
-            key=SearchKey(name="user.email"),
-            operator="=",
-            value=SearchValue(raw_value="hello@example.com"),
-        )
-        self.term5 = SearchFilter(
-            key=SearchKey(name="user.email"),
-            operator="=",
-            value=SearchValue(raw_value="hi@example.com"),
-        )
+        self.term1 = ["equals", ["user.email", "foo@example.com"]]
+        self.term2 = ["equals", ["user.email", "bar@example.com"]]
+        self.term3 = ["equals", ["user.email", "foobar@example.com"]]
+        self.term4 = ["equals", ["user.email", "hello@example.com"]]
+        self.term5 = ["equals", ["user.email", "hi@example.com"]]
 
     def test_simple(self):
-        assert parse_search_query("user.email:foo@example.com OR user.email:bar@example.com") == [
-            SearchBoolean(left_term=self.term1, operator="OR", right_term=self.term2)
-        ]
+        result = get_filter("user.email:foo@example.com OR user.email:bar@example.com")
```

Comment:
More of a question, is there any reason why we don't turn these emails into variables and use them throughout tests? I doubt they'd ever change but if they did for some reason it'd be a lot of broken tests.

It'd also have a benefit of making the asserts easier to read since you don't need to store which email refers to which term in your head.

Reply:
I've been thinking about the best way to do this and I may revisit it. I was worried about hiding too much of the details in order to try and limit the "noise" in a test, but I'll look these over again and see if it can be clearer what things are so the expected result is more obvious.

Reply:
Would renaming `term1` etc. to `foo_term`, `bar_term` help? Then the names of the email address and term attribute would have something in common.